packages/boot/test/orchestration/restart-contracts.test.ts

```diff
@@ -57,43 +56,45 @@ test.serial.failing('send-anywhere', async t => {
       chainName: 'cosmoshub',
     },
   });
-  // no errors and no resolution
-  const beforeFlush = wallet.getLatestUpdateRecord();
-  t.like(wallet.getLatestUpdateRecord(), {
-    updated: 'offerStatus',
-    status: {
-      id: 'send-somewhere',
-      error: undefined,
-    },
-    numWantsSatisfied: undefined,
-    payouts: undefined,
-    result: undefined,
-  });
 
-  t.is(await flushInboundQueue(), 0);
-  t.deepEqual(wallet.getLatestUpdateRecord(), beforeFlush);
+  t.like(
+    wallet.getCurrentWalletRecord(),
+    { liveOffers: [['send-somewhere']] },
+    'live offer until we simulate the transfer ack',
+  );
 
-  t.log('restart send-anywhere');
-  await evalProposal(
-    buildProposal('@agoric/builders/scripts/testing/restart-send-anywhere.js'),
+  // TODO #9303 Error#1: replay 12: ["checkCall","[Alleged: contractState guest wrapper]","get",["localAccount"],12] vs ["doReturn",11,"[undefined]"] : length: unequal 5 vs 3
+  // t.log('restart send-anywhere');
+  // await evalProposal(
+  //   buildProposal('@agoric/builders/scripts/testing/restart-send-anywhere.js'),
+  // );
+
+  t.like(
+    wallet.getLatestUpdateRecord(),
+    {
+      updated: 'balance',
+      currentAmount: { value: [] },
+    },
+    'no offerStatus updates',
   );
 
+  // simulate ibc/MsgTransfer ack from remote chain, enabling `.transfer()` promise
+  // to resolve
+  await inboundVTransferEvent({
+    sourceChannel: 'channel-5',
+    sequence: '1',
+  });
```

Comment:
I think this is too abstract for BridgeUtils. (see comment below)

Putting a suggestion here to see how it looks:

```suggestion
  await runInbound(BridgeId.VTRANSFER, buildVTransferEvent({
    sourceChannel: 'channel-5',
    sequence: '1',
  });
```

(and drop the export for `BuildVTransferEventParams`)

If that's too clunky, `ibc-mocks.ts` could have a helper like,
```
const makeRunIbcTransfer = (queueInbound) => 
  (event: Parameters<typeof buildVTransferEvent>[0]) => queueInbound(BridgeId.VTRANSFER, buildVTransferEvent(event);
```
and this could be,
```suggestion
  await runIbcTransfer({
    sourceChannel: 'channel-5',
    sequence: '1',
  });
```